### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by tantek**
let's not hold back from explicitly naming modern web versioning attempts

***
<a href="https://pr-preview.s3.amazonaws.com/tantek/the-web-is-not-versioned/pull/5.html" title="Last updated on Jan 25, 2024, 1:00 AM UTC (714307f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/the-web-is-not-versioned/5/7dab875...tantek:714307f.html" title="Last updated on Jan 25, 2024, 1:00 AM UTC (714307f)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -70,8 +70,8 @@
 of each technical contribution to the web
 in a distributed and modular way. 
 
-Version numbers such as "Web 2.0" have been used
-for educational, communications or marketing purposes,
+Version numbers such as "Web 2.0", "Web 3.0", "Web3", or "Web5" have been used
+for educational, communications, or marketing purposes,
 which is different from versioning in the sense that software is versioned.
```
Comment (fantasai) [inline index.html:75]: These additional examples don't fit in this sentence. They belong to the next sentence.

**Review by fantasai — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (tantek) [reply index.html:75]: They've all been used " for educational, communications, or marketing purposes" (and that's being kind). They fit there. None of them uniquely deserve to be separated out.

**Comment by chrisn** [comment]
Here's another example, for Web 4.0: https://ec.europa.eu/commission/presscorner/detail/en/ip_23_3718

**Review by rhiaro — COMMENTED** [review]
We've discussed this in our Edinburgh face-to-face, and consensus in the room is to keep the explicit naming to an the older example to reduce the potential lightning rod effect

**Pull request closed without merging**